pkg/settings/setting.go

```diff
@@ -207,7 +207,7 @@ var (
 	UIFeedBackForm = NewSetting("ui-feedback-form", "")
 
 	// UIIndex depends on ui-offline-preferred, use this version of the old ember UI instead of the one contained in Rancher Manager.
-	UIIndex = NewSetting("ui-index", "https://releases.rancher.com/ui/release-2.7.2/index.html")
+	UIIndex = NewSetting("ui-index", "https://releases.rancher.com/ui/latest2/index.html")
```

Comment:
is this supposed to be `latest2` or `latest`?

Reply:
I'm assuming at some point during the old ui's lifetime there was a `latest`, but `latest2` is the correct one. we should align the name of both of these variables at some point